pipeline/backend/docker/convert.go

```diff
@@ -20,12 +21,26 @@ func toConfig(proc *types.Step) *container.Config {
 		AttachStdout: true,
 		AttachStderr: true,
 	}
+
+	if len(proc.Commands) != 0 {
+		if runtime.GOOS == "windows" {
```

Comment:
@6543 Isn't this executed on the server? So it would use the servers OS instead of the OS selected in the pipeline config?